stdio/format.c

```diff
@@ -35,21 +35,53 @@
 #define FLAG_MINUS         0x1000
 #define FLAG_FIELD_WIDTH_STAR 	0x2000
 #define FLAG_DOUBLE		   0x4000
+#define FLAG_8BIT               0x8000
+#define FLAG_16BIT              0x10000
+#define FLAG_LONG_DOUBLE        0x20000
+
+
+#define GET_UNSIGNED(number, flags, args) \
+	do { \
+		if (((flags)&FLAG_8BIT) != 0) { \
```

Comment:
clang format is confused, applies in other places too
```suggestion
		if (((flags) & FLAG_8BIT) != 0) { \
```

Reply:
You can use `/* clang-format off */` and `/* clang-format on */`